### PR TITLE
Fix description

### PR DESCRIPTION
**Pull request by arturogonzalez58**
Fix the action description

### CODE REVIEW

**Review by hojenkai — APPROVED** [review]

File: action.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: 'Ambassador Telepresence Actions'
-description: 'Telepresence combined with GitHub Actions allows you to run integration tests in your CI pipeline without having to run any dependant service. By connecting to the target Kubernetes cluster, intercepting traffic to the remote service and sending it to an instance of the service running in CI you will be able to test bugfixes, updates and features very easily.'
+description: 'Telepresence combined with GitHub Actions allows you to run integration tests in your CI pipeline.'
```
Comment (khussey) [inline action.yaml:2]: "pipeline." -> "pipeline against services running in a remote cluster."

**Review by khussey — CHANGES_REQUESTED** [review]
See comment.

File: action.yaml (hunk 1 above)
Comment (arturogonzalez58) [reply action.yaml:2]: we have character restriction, we are allowed to use only 125 characters

Comment (khussey) [reply action.yaml:2]: How about "Telepresence GitHub Actions allows you to run integration tests against services in a remote target cluster."?

**Review by khussey — APPROVED** [review]
LGTM

**Pull request merged**